test/typescript.types.ts

```diff
@@ -28,14 +29,14 @@ emptyAction();
 function onOff(on, off) {
   on(act1, () => 1)
 }
-
-const reducers = [
-  createReducer({}),
-  createReducer({}, {}),
-  createReducer<number>({ [act1.getType()]: () => 1 }, 0),
-  createReducer<boolean>(onOff, true),
-  createReducer<number>(onOff, 1)
-]
+const a1 = createReducer({});
+const reducers = combineReducers({
+  a1, 
+  a2 :createReducer({}, {}),
+  a3 :createReducer<number>({ [act1.getType()]: () => 1 }, 0),
+  a4 :createReducer<boolean>(onOff, true),
+  a5 :createReducer<number>(onOff, 1)
+});
```

Comment:
What's the advantage of combining rather than just creating them?

Reply:
It's a way to know about Reducer type compatibility between redux-act and [redux Reducer type](https://github.com/reduxjs/redux/blob/master/index.d.ts#L62)